src/gleam/int.gleam

```diff
@@ -33,7 +33,14 @@ if erlang {
   ///
   pub external fn parse(String) -> Result(Int, Nil) =
     "gleam_stdlib" "parse_int"
+}
+
+if javascript {
+  pub external fn parse(String) -> Result(Int, Nil) =
```

Comment:
We've lost the doc comment here. Can we use this pattern?

https://github.com/gleam-lang/stdlib/blob/280a4d2ee14acc358574a32645baf1c27e8bf04c/src/gleam/list.gleam#L95-L107